Qt6.5.3+ emits new warnings and causes tests to be aborted even if did
not really fail. May(?) indicate that something needs to be done better
in PyQt6 or qutebrowser, but let's just ignore it here for now.
--- a/pytest.ini
+++ b/pytest.ini
@@ -61,2 +61,4 @@
     ^QBackingStore::endPaint\(\) called with active painter; did you forget to destroy it or call QPainter::end\(\) on it\?$
+    # ...and Qt 6.5.3
+    ^QItemSelectionModel: Selecting when no model has been set will result in a no-op\.
 xfail_strict = true
